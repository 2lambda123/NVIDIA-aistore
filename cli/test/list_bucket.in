ais gen-shards --bucket $BUCKET_1 --ext .tar --template="tmp/shard-{0..4}" --fcount 1 --fsize 1KB --conc 1 --cleanup // IGNORE
ais ls $BUCKET_1
ais ls $BUCKET_1/
ais ls $BUCKET_1
ais ls ais $BUCKET_1/
ais ls cloud $BUCKET_1/ // FAIL "bucket with name "$BUCKET_1" does not exists"
ais rm bucket $BUCKET_1
