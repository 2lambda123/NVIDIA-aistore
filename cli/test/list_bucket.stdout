Name			Size	Version
tmp/shard-0.tar		2.50KiB	1
tmp/shard-1.tar		2.50KiB	1
tmp/shard-2.tar		2.50KiB	1
tmp/shard-3.tar		2.50KiB	1
tmp/shard-4.tar		2.50KiB	1
Name			Size	Version
tmp/shard-0.tar		2.50KiB	1
tmp/shard-1.tar		2.50KiB	1
tmp/shard-2.tar		2.50KiB	1
tmp/shard-3.tar		2.50KiB	1
tmp/shard-4.tar		2.50KiB	1
Name			Size	Version
tmp/shard-0.tar		2.50KiB	1
tmp/shard-1.tar		2.50KiB	1
tmp/shard-2.tar		2.50KiB	1
tmp/shard-3.tar		2.50KiB	1
tmp/shard-4.tar		2.50KiB	1
Name			Size	Version
tmp/shard-0.tar		2.50KiB	1
tmp/shard-1.tar		2.50KiB	1
tmp/shard-2.tar		2.50KiB	1
tmp/shard-3.tar		2.50KiB	1
tmp/shard-4.tar		2.50KiB	1
$BUCKET_1 bucket destroyed
